rabbitmq/tests/metrics.py

```diff
@@ -69,6 +69,27 @@
     'rabbitmq.overview.queue_totals.messages_unacknowledged.rate',
 ]
 
+# Present from 3.8
+if RABBITMQ_VERSION >= version.parse('3.8'):
+    OVERVIEW_METRICS_TOTALS.extend(
```

Comment:
It doesn't seem we have a test environment for rabbitmq 3.8. Could you please update the test environments to add 3.8 as well [here](https://github.com/DataDog/integrations-core/blob/master/rabbitmq/tox.ini)?